C++/Keyword.hpp

```diff
@@ -29,8 +29,7 @@
 
 #include "GetkwError.hpp"
 
-namespace KeyType {
-enum KeyKinds {
+enum KeyKinds : int {
```

Comment:
Don't you need a `enum class` to scope it?

Reply:
: facepalm: of course I do...